### ISSUE TITLE
About background class 

### ISSUE THREAD

**Issue by yu-xi-wang** [ZijiaLewisLu/ICCV2025-MMF-TAS#2]
Hi,

Thank you for this excellent work! I would greatly appreciate some clarification regarding the details of how PGNet operates.

Specifically, I’m trying to understand how the frame-wise background predictions are generated. In the paper, it is stated that the attention map **Δ** is treated as the prediction logits, with a shape of **[L, A]**, where *L* is the video length. Does *A* include the background class? If so, does that mean that when initializing the ARG, there is a background action-prototype node for each task?

Thank you very much for your time and help.

Best regards,  
Yuxi

**Comment by ZijiaLewisLu** [comment]
Hi,

Thank you for your patience. Yes, *A* include a background class and each task always has a background-prototype. 

Zijia
